_posts/2020-03-18-http-security-headers.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: Staying un-pwned with HTTP Security Headers
+author: massimo
+excerpt: >
+  Your fancy front-end framework won't print unescaped HTML. But what if it does after the next update? Well, that's why you have a WAF. What if there's a way to circumvent it? There's always a what if. This is not a comprehensive list of all security headers. These are the bare minimum every non-trivial site should use.
```

Comment:
I would simplify this somehow. Had to look up WAF. :)    It said Women in The Air Force. Is firewall the next layer after escaping html? Next, you're telling what this blog is not about. Rather explain what this is about?

Your fancy front-end framework won't print unescaped HTML. But what if it does after the next update? To protect your app, you need security layers. Like an onion, each layer means more tears to the attacker. Security headers form one of these layers. This blog presents a list of security headers, which should be used by every non-trivial site.

Reply:
Looking things up is good :) But I'll add an explanation about the WAF acronym. I think you're correct about the order of the layers but I'd still like to keep the WAF as an example since many might regard it as a silver bullet.